bin/ycsb

```diff
@@ -120,12 +122,24 @@ def usage():
 
     return output.getvalue()
 
-def check_output(cmd):
-    p = subprocess.Popen(cmd, stdout=subprocess.PIPE)
-    stdout, _ = p.communicate()
-    if p.returncode:
-        raise subprocess.CalledProcessError(p.returncode, cmd)
-    return stdout
+# From https://gist.github.com/edufelipe/1027906
```

Comment:
Maybe it makes sense to pull straight from the source instead?

https://github.com/python/cpython/blob/2.7/Lib/subprocess.py#L545

It would be under the license here: https://github.com/python/cpython/blob/2.7/LICENSE